src/commands/toolchain.rs

```diff
@@ -42,6 +50,20 @@ pub struct UninstallCommand {
 #[derive(Debug, Parser)]
 pub struct ListRevisionsCommand {}
 
+#[derive(Debug, Deserialize, Parser)]
+pub struct ExportCommand {
+    /// Toolchain to export, [possible values: latest, beta-1, beta-2, beta-3, beta-4, nightly].
+    /// The default toolchain will be exported if name isn't specified
+    pub name: Option<String>,
+    /// Channel to export, [possible values: latest-YYYY-MM-DD, nightly-YYYY-MM-DD, beta-1, beta-2, beta-3, beta-4].
```

Comment:
Does this also have a default if not specified?

Reply:
Just as described here "The default toolchain will be exported if name isn't specified" , the default toolchain by `fuelup default` would be used for.

![image](https://github.com/FuelLabs/fuelup/assets/25278203/7f7c2944-0b37-4f9b-ba11-5a41a01984df)